media-gfx/sane_backends/sane_backends-1.0.30.recipe

```diff
@@ -18,12 +18,12 @@ HOMEPAGE="http://www.sane-project.org"
 COPYRIGHT="David Mosberger-Tang, Andy Beck"
 LICENSE="GNU LGPL v2"
 REVISION="1"
-SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/54f858b20a364fc35d820df935a86478/sane-backends-$portVersion.tar.gz"
-CHECKSUM_SHA256="aa027b4e5f59849cd41b8c26d54584cf31fffd986049019be6ad4140e11ea8ed"
+SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/c3dd60c9e054b5dee1e7b01a7edc98b0/sane-backends-$portVersion.tar.gz"
+CHECKSUM_SHA256="3f5d96a9c47f6124a46bb577c776bbc4896dd17b9203d8bfbc7fe8cbbcf279a3"
 SOURCE_DIR="sane-backends-$portVersion"
 PATCHES="sane_backends-$portVersion.patchset"
 
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
Parse errors on gcc2, probably could be fixed, question is, do you want to take the time to fix them? :)